lib/fluri.dart

```diff
@@ -199,16 +199,71 @@ class FluriMixin {
     _uri = _uri.replace(queryParameters: queryParameters);
   }
 
+  /// The URI query parameters with support for multi-value params.
+  Map<String, List<String>> get queryParametersAll => _uri.queryParametersAll;
+  set queryParametersAll(Map<String, Iterable<String>> queryParameters) {
```

Comment:
The existing `queryParameters` getter/setter are typed `Map<String, String>`. My first approach to this was relaxing the typing to `dynamic` and then doing the type checks in line and throwing `ArgumentError`s, but I ended up going this route because it mimics how the `Uri` class accomplished this and it retains the static typing.